server.go

```diff
@@ -707,9 +714,9 @@ func (s *Server) multicastResponse(msg *dns.Msg, ifIndex int) error {
 			wcm.IfIndex = ifIndex
 			s.ipv6conn.WriteTo(buf, &wcm, ipv6Addr)
 		} else {
-			for ifi := range s.ifaces {
-				wcm.IfIndex = s.ifaces[ifi].Index
-				s.ipv6conn.WriteTo(buf, &wcm, ipv4Addr)
```

Comment:
This was a pretty big typo, using ipv4Addr instead of ipv6Addr

Reply:
Good catch! :+1: